.github/workflows/release-please.yml

```diff
@@ -48,3 +53,46 @@ jobs:
               config/webhook/certificate.yaml
               config/rendered/release.yaml
               config/samples/end-to-end.yaml
+
+  build-oci:
+    needs: release-please
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-please.outputs.release_created }}
```

Comment:
Perhaps this check isn't needed if we want to publish a new image for every push to main, not only on tag creation @AlexsJones

Reply:
nah we dont need too